topicgenerator/src/main/java/io/dockstore/topicgenerator/client/cli/TopicGeneratorClient.java

```diff
@@ -44,21 +49,12 @@
 import org.slf4j.LoggerFactory;
 
 public class TopicGeneratorClient {
-    // Headers for the input CSV file
-    public static final String TRS_ID_CSV_HEADER = "trsId";
-    public static final String VERSION_CSV_HEADER = "version";
-    // Headers for the output CSV file
-    public static final String AI_TOPIC_CSV_HEADER = "aiTopic";
-    public static final String PROMPT_TOKENS = "promptTokens"; // Number of tokens in prompt
-    public static final String COMPLETION_TOKENS = "completionTokens"; // Number of tokens in response
-    public static final String FINISH_REASON = "finishReason"; // The reason that the response stopped.
-    public static final String IS_TRUNCATED = "isTruncated"; // Whether the descriptor file content truncated because it exceeded the token maximum
     public static final String OUTPUT_FILE_PREFIX = "generated-topics";
-    protected static final String[] INPUT_CSV_HEADERS = { TRS_ID_CSV_HEADER, VERSION_CSV_HEADER };
-    protected static final String[] OUTPUT_CSV_HEADERS = { TRS_ID_CSV_HEADER, VERSION_CSV_HEADER, IS_TRUNCATED, PROMPT_TOKENS, COMPLETION_TOKENS, FINISH_REASON, AI_TOPIC_CSV_HEADER };
     private static final Logger LOG = LoggerFactory.getLogger(TopicGeneratorClient.class);
-    // Using GPT 3.5-turbo-16k because it's cheaper and faster than GPT4, and it takes more tokens (16k). GPT4 seems to cause a lot of timeouts.
-    private static final ModelType AI_MODEL = ModelType.GPT_3_5_TURBO_16K;
+    // Using GPT 3.5-turbo because it's cheaper and faster than GPT4, and it takes more tokens (16k). GPT4 seems to cause a lot of timeouts.
```

Comment:
Changed the AI model from GPT-3.5-turbo-16k to GPT-3.5-turbo because the former is now considered a legacy model since the GPT-turbo-3.5 model now has a default context window of 16k tokens. Can see more info about the models here https://platform.openai.com/docs/models/gpt-3-5-turbo.

The latest GPT-3.5 Turbo model supposedly has a higher accuracy at responding in requested formats